Argo/Types/DecodeError.swift

```diff
@@ -13,3 +13,30 @@ extension DecodeError: CustomStringConvertible {
     }
   }
 }
+
+extension DecodeError: Hashable {
+  public var hashValue: Int {
+    switch self {
+    case let .TypeMismatch(expected: expected, actual: actual):
+      return expected.hashValue ^ actual.hashValue
+    case let .MissingKey(string): return string.hashValue
```

Comment:
I think since the `TypeMismatch` return statement is broken to a new line we should probably stay consistent with these other two.